src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -2,21 +2,51 @@
 Copyright (c) 2018 Patrick Massot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot, Johannes Hölzl
-
-Continuous linear functions -- functions between normed vector spaces which are bounded and linear.
 -/
 import analysis.normed_space.multilinear
 
+/-!
+# Continuous linear maps
+
+This file defines a class stating that a map between normed vector spaces is (bi)linear and
+continuous.
+Instead of asking for continuity, the definition takes the equivalent condition that `∥f x∥` is
+bounded by a multiple of `∥x∥`. Hence the "bounded" in the name refers to `∥f x∥/∥x∥` rather than
+`∥f x∥` itself.
+
+## Main declarations
+
+* `is_bounded_linear_map`: Class stating that a map `f : E → F` is linear and continuous, but
+  through the simpler to provide statement that `∥ f x ∥` is bounded by a multiple of `∥ x ∥`.
+  Effectively an unbundled version of `continuous_linear_map`, albeit the name disparity.
+* `is_bounded_bilinear_map`! Class stating that a map `f : E × F → G` is bilinear and continuous,
+  but through the simpler to provide statement that `∥f (x, y)∥` is bounded by a multiple of
+  `∥x∥ * ∥y∥`
+* `is_bounded_bilinear_map.linear_deriv`: Derivative of a continuous bilinear map as a linear map.
+* `is_bounded_bilinear_map.deriv`: Derivative of a continuous bilinear map as a continuous linear
+  map. The proof that it is indeed the derivative is `is_bounded_bilinear_map.has_fderiv_at` in
+  `analysis.calculus.fderiv`.
+* `linear_map.norm_apply_of_isometry`: A linear isometry preserves the norm.
+
+## Notes
+
+The main use of this file is `is_bounded_bilinear_map`. The file `analysis.normed_space.multilinear`
+already expands the theory of multilinear maps, but the `2`-variables case is sufficiently simpler
+to currently deserve its own treatment.
+
+`is_bounded_linear_map` should in most cases be replaced by a bundled `continuous_linear_map`. The
```

Comment:
Also, to nitpick a little here, "replaced" suggests to my ear that there are occurrences of `is_bounded_linear_map` currently where switching over to `continuous_linear_map` is a TODO.  I don't think that's the case -- there's basically no use of `is_bounded_linear_map` in the library.  So what we want to say is, rather, addressed to future users:  that `continuous_linear_map` should be preferred to `is_bounded_linear_map`.